tools/mitohifi/mitohifi.xml

```diff
@@ -0,0 +1,84 @@
+    <requirements>
+        <requirement type="package" version="2.9.0">blast</requirement>
+        <requirement type="package" version="1.78">biopython</requirement>
+        <requirement type="package" version="1.2.2">pandas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        bash '$__tool_directory__/run.sh' -c '$contigs' -f '$mitogenome_fa' -g '$mitogenome_gb' -t \${GALAXY_SLOTS:-1} -o $organism '$output1'
+    ]]></command>
+    <inputs>
+        <param type="data" name="contigs" format="fasta" label="Contig file in fasta format" />
+        <param type="data" name="mitogenome_fa" format="fasta"  label="Close-related species mitogenome in fasta format" />
+        <param type="data" name="mitogenome_gb" format="genbank"  label="Close-related species mitogenome in genbank format" />
+        <param type="select" name="organism" label="Organism genetic code following NCBI table">
+            <option value="1">1. The Standard Code</option>
+            <option value="2">2. The Vertebrate Mitochondrial</option>
+            <option value="3">3. The Yeast Mitochondrial Code</option>
+            <option value="4">4. The Mold, Protozoan, and Coelenterate Mitochondrial Code and the Mycoplasma/Spiroplasma Code</option>
+            <option value="5">5. The Invertebrate Mitochondrial Code</option>
+            <option value="6">6. The Ciliate, Dasycladacean and Hexamita Nuclear Code</option>
+            <option value="9">9. The Echinoderm and Flatworm Mitochondrial Code</option>
+            <option value="10">10. The Euplotid Nuclear Code</option>
+            <option value="11">11. The Bacterial, Archaeal and Plant Plastid Code</option>
+            <option value="12">12. The Alternative Yeast Nuclear Code</option>
+            <option value="13">13. The Ascidian Mitochondrial Code</option>
+            <option value="14">14. The Alternative Flatworm Mitochondrial Code</option>
+            <option value="16">16. Chlorophycean Mitochondrial Code</option>
+            <option value="21">21. Trematode Mitochondrial Code</option>
+            <option value="22">22. Scenedesmus obliquus Mitochondrial Code</option>
+            <option value="23">23. Thraustochytrium Mitochondrial Code</option>
+            <option value="24">24. Pterobranchia Mitochondrial Code</option>
+            <option value="25">25. Candidate Division SR1 and Gracilibacteria</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="output1" format="fasta" />
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="contigs" value="test.fasta" />
+            <param name="mitogenome_fa" value="NC_016067.1.fasta" />
+            <param name="mitogenome_gb" value="NC_016067.1.gb" />
+            <output name="output1" file="mitogenome.fasta" />
+        </test>
+    </tests>
+
+    <help><![CDATA[
+Usage: '$0 -c contigs.fasta -f close-related_mitogenome.fasta -g close-related_mitogenome.gb -t threads'
```

Comment:
I would remove this help, this is already covered in the form. A more meaningful help would be nice.

e.g.

```
MitoHiFi circularises, cuts and annotates the mitogenome from contigs assembled with PacBio HiFi reads and softwares such as HiCanu or Hifiasm.

The dissemination of high-quality long reads - such as PacBio HiFi - makes the assembly of high-quality mitogenome straight forward. Because of the circular nature of the molecule, however, the mitocontig is usually assembled redundantly resulting in multiple-copy mitogenome-contigs. This pipeline was developed to finalise the assembly and annotation of the mitogenome by:

(i) finding it among complete genome assembled contigs
(ii) circularising and cutting it to represent only one copy of the circular molecule and
(iii) producing an annotation and presenting it in fasta and genbank format.

    This pipeline can also be ran starting from polished PacBio CLR contigs or ONT contigs. You must be sure your contigs are well polished.
```